### DISCUSSION TITLE
"Request not allowed" when trying to download export

### DISCUSSION THREAD

**Discussion by rkrug** [retorquere/zotero-better-bibtex#2587] [General]
Hi

OS: MacOS 13.4.1 Ventura
Zotero: 6.0.26

when trying the "Download Better BibTex export...", I get an "Request not allowed" error. I tried to download a group as well as a collection. Examples I tried are (copied from Zotero):

For a group:
`http://127.0.0.1:23119/better-bibtex/collection?/0/8CV58ZVD.biblatex`

For a collection
`http://127.0.0.1:23119/better-bibtex/export/collection?/1/Data%20Management.biblatex`
`http://127.0.0.1:23119/better-bibtex/export/collection?/1/TU48LUCD.biblatex`

Hm.

I upgraded to BetterBibTex 26 July 2023 and Nothing happens when I click on "Download Better BibTex export...".

Zotero: 6.0.26

Is this a problem on y side?

**Comment by retorquere** [comment]
Did you try to download using a browser? Zotero does not allow browsers to use the http api.

**Reply by rkrug** [reply]
Yes - I did. OK. That explains the first part. But how do I get the URL after upgrade to Better BibTex from the 26 July 2023? The dialog showing the URL seems to be broken.

**Reply by retorquere** [reply]
For that you'll need to open a new issue.

**Reply by rkrug** [reply]
Will do so.
